### PR TITLE
Add support to aws_ec2_elb_lb

### PR DESCRIPTION
**Pull request by samvarankashyap**
Summary:
b02984b4 Added integration test for aws_ec2_elb_lb
db369453 Updated documentation and examples for aws-ec2-elb-lb
c9d343c8 Updated playbooks to support aws_ec2_elb_lb
a4661287 Updated schema to support aws_ec2_elb_lb

fixes: #826 #824 
to be merged after #947

### CODE REVIEW

**Comment by p3ck** [comment]
[test]

**Comment by samvarankashyap** [comment]
@p3ck This PR needs to be merged after #947 . 
I need to update the tests too.

**Comment by p3ck** [comment]
@samvarankashyap ok, I wanted to review the failures but the job rotated out.  That is why I ran it again.

**Comment by samvarankashyap** [comment]
@p3ck @14rcole Needs review !

File: config/Dockerfiles/tests.d/aws/06_aws_ec2_elb_lb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -xe
+
+# Verify the aws ec2 elb lb provisioning
+# distros.exclude: centos7fedora28
```
Comment (p3ck) [inline config/Dockerfiles/tests.d/aws/06_aws_ec2_elb_lb]: I think you're missing a space here.

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**